app/controllers/ctc/questions/stimulus_three_controller.rb

```diff
@@ -1,16 +1,10 @@
 module Ctc
   module Questions
-    class StimulusOneController < QuestionsController
+    class StimulusThreeController < QuestionsController
       include AuthenticatedCtcClientConcern
 
       layout "intake"
 
```

Comment:
I guess we'll have to remember to add a skip function later--- or could we do it depending on if they set an amount on the stimulus-payments page?

Reply:
but I guess we dont have the stimulus payments page adjusted for EIP-3